spec/spec_helper.rb

```diff
@@ -69,6 +69,10 @@
   # rspec-rails.
   config.infer_base_class_for_anonymous_controllers = false
 
+  # TODO - remove this line after Spree upgrade. Current Spree specs do not have `type: :controller` definitions.
+  # This is a workaround to pass tests in Travis i.e. https://travis-ci.org/openfoodfoundation/openfoodnetwork/jobs/235601640
+  config.infer_spec_type_from_file_location!
+
```

Comment:
Sorry @jeronimo, I'm just trying to document the issues (for my own information as much as anyone else's). I'm definitely not suggesting that you should have been aware of work that I was doing on a totally different branch!! 🙂